CHANGELOG.md

```diff
@@ -9,7 +9,39 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
 
 _The old changelog can be found in the `release-2.6` branch_
 
-# Changes Since v3.0.3
+# Changes Since v3.1.0
+
+# v3.1.0 - [2019.02.08]
+
+## New Commands
+  - Introduced the `oci` command group to support a new OCI compliant variant of the Singularity runtime:
+    - `attach` Attach console to a running container process
+    - `create` Create a container from a bundle directory
+    - `delete` Delete container
+    - `exec`   Execute a command within container
+    - `kill`   Kill a container
+    - `mount`  Mount create an OCI bundle from SIF image
+    - `pause`  Suspends all processes inside the container
+    - `resume` Resumes all processes previously paused inside the container
+    - `run`    Create/start/attach/delete a container from a bundle directory
+    - `start`  Start container process
+    - `state`  Query state of a container
+    - `umount` Umount delete bundle
+    - `update` Update container cgroups resources
+  - Added `cache` command group to inspect and manage cached files
+    - `clean` Clean your local Singularity cache
+    - `list`  List your local Singularity cache
+
```

Comment:
Wheres the `apps` command? I think that should be here...

Reply:
??
```
vagrant@ubuntu-bionic:~/go/src/github.com/sylabs/singularity$ singularity help cache
Manage your local singularity cache

Usage:
  singularity cache <subcommand>

Description:
  Manage your local singularity cache. There are 3 types of cache; library, oci, and blob.
  You can list/clean using the spicific types.

Options:
  -h, --help   help for cache

Available Commands:
  clean       Clean your local Singularity cache
  list        List your local Singularity cache

Examples:
  All group commands have their own help output:

  $ singularity cache
  $ singularity cache --help


For additional help or support, please visit https://www.sylabs.io/docs/
```

Reply:
^ don't see `apps` command

Reply:
Sorry, `apps` command here:

```
$ singularity 
Usage:
  singularity [global options...] <command>

Available Commands:
  apps        apps <image path>
  build       Build a new Singularity container
  cache       Manage your local singularity cache
  capability  Manage Linux capabilities on containers
  exec        Execute a command within container
  inspect     Display metadata for container if available
  instance    Manage containers running in the background
  key         Manage OpenPGP key stores
  oci         Manage OCI containers
  pull        Pull a container from a URI
  push        Push a container to a Library URI
  run         Launch a runscript within container
  run-help    Display help for container if available
  search      Search the library
  shell       Run a Bourne shell within container
  sign        Attach cryptographic signatures to container
  test        Run defined tests for this particular container
  verify      Verify cryptographic signatures on container
  version     Show application version

```

Reply:
I was thinking that was covered by this:
`- Fixed several regressions and omissions in [SCIF](https://sci-f.github.io/) support`
Because `apps` used to be command in 2.x and so it's not really new.  Including it just fixes an omission/regression.